modules/admin_manual/pages/configuration/server/background_jobs_configuration.adoc

```diff
@@ -77,58 +60,44 @@ executing:
 
 NOTE: You have to make sure that PHP is found by Cron; hence why we’ve deliberately added the full path.
 
-Please refer to https://linux.die.net/man/1/crontab[the crontab man page]
-for the exact command syntax if you don’t want to have it run every minute.
+Please refer to https://linux.die.net/man/1/crontab[the crontab man page] for the exact command syntax if you don’t want to have it run every minute.
 
-NOTE: There are other methods to invoke programs by the system regularly, e.g., 
-https://wiki.archlinux.org/index.php/Systemd/Timers[systemd timers]
+NOTE: There are other methods to invoke programs by the system regularly, e.g., https://wiki.archlinux.org/index.php/Systemd/Timers[systemd timers]
 
-[[webcron]]
 === Webcron
 
-By registering your ownCloud `cron.php` script address as an external
-webcron service (for example, http://www.easycron.com/[easyCron]), you
-ensure that background jobs are executed regularly. To use this type of
-service, your external webcron service must be able to access your
-ownCloud server using the Internet. For example:
+By registering your ownCloud `cron.php` script address as an external webcron service (for example, http://www.easycron.com/[easyCron]), you ensure that background jobs are executed regularly. 
+To use this type of service, your external webcron service must be able to access your ownCloud server using the Internet. 
+For example:
 
 ----
 URL to call: http[s]://<domain-of-your-server>/owncloud/cron.php
```

Comment:
`cron.php` --> `cron`

Reply:
This doesn't make sense here. How can a command-line binary be called via a web request?

Reply:
you are totally right,
I am in sorry, copy paste error. this should be `cron` only (no .php extension) 🤦‍♂